INN/umbrella-mwcir
# Article updates

## Changes

This pull request makes the following changes:

- Copies the hero styles from MTFP to reduce hero image size
https://github.com/INN/umbrella-mtfp/blob/52e4d429670bb2310ec33dd2bc2b970da82c86e4/wp-content/themes/mtfp/less/single.less#L88-L92

Before:
![screencapture-investigatemidwest-org-2016-12-21-crisis-in-our-wells-iowas-private-well-water-often-goes-untested-presenting-unknown-health-risks-2020-02-18-10_03_53](https://user-images.githubusercontent.com/18353636/74748424-7b62b700-5236-11ea-8d53-7082ccf710bd.png)

After:
![screencapture-mwcir-test-2016-12-21-crisis-in-our-wells-iowas-private-well-water-often-goes-untested-presenting-unknown-health-risks-2020-02-18-10_04_07](https://user-images.githubusercontent.com/18353636/74748419-77369980-5236-11ea-8a69-aafb2aef45fe.png)

## Why



For #28 

## Testing/Questions

Features that this PR affects:

- Article page styling


Questions that need to be answered before merging:


Steps to test this PR:

1. View an article with a big hero image
2. Make sure the size is reduced and matches the size on MTFP
